src/field_theory/fixed.lean

```diff
@@ -73,8 +73,7 @@ lemma linear_independent_smul_of_linear_independent {s : finset F} :
   linear_independent (fixed_points G F) (λ i : (↑s : set F), (i : F)) →
   linear_independent F (λ i : (↑s : set F), mul_action.to_fun G F i) :=
 begin
-  haveI : is_empty ((∅ : finset F) : set F) := ⟨subtype.prop⟩,
-  refine finset.induction_on s (λ _, linear_independent_empty_type)
+  refine finset.induction_on s (λ _, linear_independent_empty_type $ λ ⟨x⟩, x.2)
```

Comment:
This is more bad merge

Reply:
Ah! Sorry, this PR is from so long ago I mistakenly decided this change was on purpose. :-)

Reply:
It might almost have been easier to just undo the merge commit and start over, although I think you probably have got them all now.